yarn.lock

```diff
@@ -8258,7 +8258,7 @@ __metadata:
     node-fetch: ^2.6.7
     nunjucks: ^3.2.3
     octokit: ^2.0.0
-    octokit-plugin-create-pull-request: ^3.10.0
+    octokit-plugin-create-pull-request: 4.1.1
```

Comment:
There is an problem with 4.1.2 version of octokit-plugin-create-pull-request as workaround hardcoded to only use 4.1.1 [link to issue](https://github.com/gr2m/octokit-plugin-create-pull-request/issues/124)